Extensions/Multiplayer/multiplayertools.ts

```diff
@@ -976,6 +995,34 @@ namespace gdjs {
       focusOnGame(runtimeScene);
     };
 
+    const sendSessionInformation = (runtimeScene: gdjs.RuntimeScene) => {
+      const lobbiesIframe = multiplayerComponents.getLobbiesIframe(
+        runtimeScene
+      );
+      if (!lobbiesIframe || !lobbiesIframe.contentWindow) {
+        // Cannot send the message if the iframe is not opened.
+        return;
+      }
+
+      // @ts-ignore - cordova param is added by Cordova.
```

Comment:
Should this be factored? It's used in runtimegame.ts, in this file and in the other